tests/lib/fakestore/refresh/refresh.go

```diff
@@ -235,7 +250,7 @@ func makeNewSnapRevision(orig, new *info, targetDir string, db *asserts.Database
 	}
 	a, err = db.Sign(asserts.SnapRevisionType, headers, nil, systestkeys.TestStoreKeyID)
 	if err != nil {
-		return err
+		return "", err
 	}
 
 	return writeAssert(a, targetDir)
```

Comment:
wasn't it less disruptive to change this to:

    _, err := writeAssert(a, targedtDir)
    return err

instead of changing the return signature here?